crud-module/index.js

```diff
@@ -0,0 +1,138 @@
+        mkdirp(folderPath + "/client/services");
+        mkdirp(folderPath + "/client/views");
+        mkdirp(folderPath + "/server");
+        mkdirp(folderPath + "/server/config");
+        mkdirp(folderPath + "/server/controllers");
+        mkdirp(folderPath + "/server/models");
+        mkdirp(folderPath + "/server/policies");
+        mkdirp(folderPath + "/server/routes");
+        mkdirp(folderPath + "/tests");
+        mkdirp(folderPath + "/tests/client");
+        mkdirp(folderPath + "/tests/e2e");
+        mkdirp(folderPath + "/tests/server");
+    },
+
+    getPrompts: function() {
+        var done = this.async();
+
+        var prompts = [{
+            type: 'confirm',
+            name: 'addAngularModule',
+            message: 'Would you like to generate the Angular Module?',
+            default: true
+        }, {
+            type: 'confirm',
+            name: 'addExpressModule',
+            message: 'Would you like to generate the Express Module?',
+            default: true
+        }];
+
+        this.prompt(prompts, function(props) {
+            this.addAngularModule = props.addAngularModule;
+            this.addExpressModule = props.addExpressModule;
+
+            done();
+        }.bind(this));
+    },
+
+    copyTemplates: function() {
+
+        var files = {
+            'module.client.module.js' : folderPath  + "/client/" + this.moduleName + ".client.module.js",
+            'module.client.config.js' : folderPath + "/client/config/" + this.moduleName + ".client.config.js",
+            'module.client.controller.js' : folderPath  + "/client/controllers/" + this.moduleName + ".client.controller.js",
+            'module.client.routes.js' : folderPath  + "/client/config/" + this.moduleName + ".client.routes.js",
+            'module.client.service.js' : folderPath  + "/client/services/" + this.moduleName + ".client.service.js",
+            'views/create.client.view.html' : folderPath  + "/client/views/create.client.view.html",
+            'views/edit.client.view.html' : folderPath  + "/client/views/edit.client.view.html",
+            'views/list.client.view.html' : folderPath  + "/client/views/list.client.view.html",
+            'views/view.client.view.html' : folderPath  + "/client/views/view.client.view.html",
+            'module.server.config.js' : folderPath  + "/server/config/" + this.moduleName + ".server.config.js",
+            'module.server.controller.js' : folderPath  + "/server/controllers/" + this.moduleName + ".server.controller.js",
+            'module.server.model.js' : folderPath  + "/server/models/" + this.moduleName + ".server.model.js",
+            'module.server.policy.js' : folderPath  + "/server/policies/" + this.moduleName + ".server.policy.js",
+            'module.server.routes.js' : folderPath  + "/server/routes/" + this.moduleName + ".server.routes.js"
+        };
+
+        for(var template in files) {
+            this.fs.copyTpl(
```

Comment:
You mean the variable names or the manner in which things are copied. If I were to remove this object style loop. The same this.fs.copyTpl etc etc would appear 10+ times. Also this way, it's much easier to see which file is copied to which location.